server/templates/placelist.html

```diff
@@ -15,22 +15,22 @@
 #}
 {% extends 'base.html' %}
 
-{% set main_id = 'sitemap' %}
-{% set title = 'Sitemap' %}
+{% set main_id = 'placelist' %}
+{% set title = 'Place List' %}
 
 {% block head %}
   <link rel="stylesheet" href={{url_for('static', filename='css/static.min.css', t=config['GAE_VERSION'])}} />
 {% endblock %}
 
 {% block content %}
   <div class="container">
-    <h1>Sitemap</h1>
+    <h1>Place List</h1>
```

Comment:
Nit: maybe call this "Places in the Data Commons Graph"

Reply:
Done